tools/sggh/tools.go

```diff
@@ -23,24 +23,39 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 
 func PrepareCommand(ctx context.Context) error {
 	hostOS := runtime.GOOS
+	ext := "tar.gz"
+	if hostOS == sgtool.Darwin {
+		hostOS = "macOS"
+		ext = "zip"
+	}
 	hostArch := runtime.GOARCH
 	binDir := sg.FromToolsDir(name, version)
 	binary := filepath.Join(binDir, name)
 	binURL := fmt.Sprintf(
-		"https://github.com/cli/cli/releases/download/v%s/gh_%s_%s_%s.tar.gz",
+		"https://github.com/cli/cli/releases/download/v%s/gh_%s_%s_%s.%s",
 		version,
 		version,
 		hostOS,
 		hostArch,
+		ext,
 	)
-	if err := sgtool.FromRemote(
-		ctx,
-		binURL,
+
```

Comment:
pls avoid adding blank lines (everyone has different taste for blank lines so to be consistent, try and limit)

Reply:
sure, I can get rid of it for you 😄